modules/installation-vsphere-config-yaml.adoc

```diff
@@ -103,48 +103,24 @@ imageContentSources: <18>
-which are functions that the cluster does not perform when you
-use user-provisioned infrastructure. You must manually deploy worker
-machines for the cluster to use before you finish installing {product-title}.
-<5> The number of control plane machines that you add to the cluster. Because
-the cluster uses this values as the number of etcd endpoints in the cluster, the
-value must match the number of control plane machines that you deploy.
+<4> You must set the value of the `replicas` parameter to `0`. This parameter controls the number of workers that the cluster creates and manages for you, which are functions that the cluster does not perform when you use user-provisioned infrastructure. You must manually deploy worker machines for the cluster to use before you finish installing {product-title}.
+<5> The number of control plane machines that you add to the cluster. Because the cluster uses this values as the number of etcd endpoints in the cluster, the value must match the number of control plane machines that you deploy.
 <6> The cluster name that you specified in your DNS records.
 <7> The fully-qualified hostname or IP address of the vCenter server.
-<8> The name of the user for accessing the server. This user must have at least
-the roles and privileges that are required for
-link:https://github.com/vmware-archive/vsphere-storage-for-kubernetes/blob/master/documentation/vcp-roles.md[static or dynamic persistent volume provisioning]
-in vSphere.
+<8> The name of the user for accessing the server.
 <9> The password associated with the vSphere user.
 <10> The vSphere datacenter.
 <11> The default vSphere datastore to use.
 <12> Optional: For installer-provisioned infrastructure, the absolute path of an existing folder where the installation program creates the virtual machines, for example, `/<datacenter_name>/vm/<folder_name>/<subfolder_name>`. If you do not provide this value, the installation program creates a top-level folder in the datacenter virtual machine folder that is named with the infrastructure ID. If you are providing the infrastructure for the cluster, omit this parameter.
-<13> Optional: For installer-provisioned infrastructure, the absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/<optional_nested_resource_pool_name>`. If you do not specify a value, resources are installed in the root of the cluster `/example_datacenter/host/example_cluster/Resources`.
+<13> Optional: For installer-provisioned infrastructure, the absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/ optional_nested_resource_pool_name>`. If you do not specify a value, resources are installed in the root of the cluster `/ example_datacenter/host/example_cluster/Resources`.
```

Comment:
```suggestion
<13> Optional: For installer-provisioned infrastructure, the absolute path of an existing resource pool where the installation program creates the virtual machines, for example, `/<datacenter_name>/host/<cluster_name>/Resources/<resource_pool_name>/<optional_nested_resource_pool_name>`. If you do not specify a value, resources are installed in the root of the cluster `/example_datacenter/host/example_cluster/Resources`.
```
I'm not sure why the `<` got removed here, but it should be re-added. there are a couple of incorrect spaces as well.